test/challtestsrv.py

```diff
@@ -1,5 +1,5 @@
-import urllib2
 import json
+import requests
```

Comment:
Was there a reason you included this diff and the ones in `test/chisel2.py` and `test/entrypoint.sh` in this branch? They seem like sensible cleanups but I'm not sure I understand the connection to the slow query integration tests.

Reply:
Oh right, that's how these changes made it in! I was working on a python3 branch around the same time and got some cross-contamination. Will clean up.